src/shared/Microsoft.Git.CredentialManager/Authentication/MicrosoftAuthentication.cs

```diff
@@ -205,6 +234,15 @@ private async Task<IPublicClientApplication> CreatePublicClientApplicationAsync(
                 appBuilder.WithParentActivityOrWindow(() => new IntPtr(hWndInt));
             }
 
+            // On Windows 10 & .NET Framework try and use the WAM broker
+            if (enableBroker && PlatformUtils.IsWindows10())
+            {
+#if NETFRAMEWORK
+                appBuilder.WithExperimentalFeatures();
+                appBuilder.WithWindowsBroker();
+#endif
+            }
+
```

Comment:
for `StorageCreationPropertiesBuilder(cacheFileName, cacheDirectory, clientId)` just use `StorageCreationPropertiesBuilder(cacheFileName, cacheDirectory)`. The `clientId` param is only needed for situations where you want to monitor for accounts created / deleted via an event. This is used by VS to update their UI whenver an app (e.g. GCM) adds or removes an account. It's not being used by GCM.

Reply:
Updated to remove this use of the deprecated ctor.